Adaptors/MongoDB/src/IMongoQueryableExt.cs

```diff
@@ -40,4 +43,36 @@ internal static class IMongoQueryableExt
       }
     }
   }
+
+  public static IOrderedMongoQueryable<T> OrderByList<T>(this IMongoQueryable<T>            queryable,
+                                                         List<Expression<Func<T, object?>>> orderFields,
```

Comment:
```suggestion
                                                         ICollection<Expression<Func<T, object?>>> orderFields,
```
I think there is no need for a List specifically. You could at least take a `ICollection`, and maybe a `IEnumerable` even.

Reply:
ICollection is effectively good. We will have a multiple iteration problem with IEnumerable due to the first and skip.